source/reference/minio-mc/mc-support-perf.rst

```diff
@@ -0,0 +1,208 @@
+===================
+``mc support perf``
+===================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support perf
+
+Description
+-----------
+
+Use the :mc-cmd:`mc support perf` command to analyze the performance of an object, network, or drive.
+
+The resulting test can provide general guidance of deployment performance under S3 ``GET`` and ``PUT`` requests.
```

Comment:
And, more importantly, where any potential bottlenecks might be. Generally Network -> Drives -> S3 API in order from most to least likely.